### ISSUE TITLE
[Bug]: [a11y] The non-wallpaper thumbnail is not announced with Talkback

### ISSUE THREAD

**Issue by AdinaPetridean** [mozilla-mobile/fenix#27129]
### Steps to reproduce

1. Turn TalkBack (or the available screen reader) on.
2. Launch the Firefox application.
3. Go to 3dot menu- Settings- Homepage- Wallpapers using TB gestures.
4. Tap on the non-wallpaper thumbnail and notice what talkback announces.

### Expected behaviour

The screen reader announces its title and the available actions (double tap to activate, double tap and hold to long press).

### Actual behaviour

The non-wallpaper thumbnail is not announced with Talkback.

### Device name

Xperia 10     

### Android version

 Android 12

### Firefox release type

Firefox Nightly

### Firefox version

Nightly 107.0a11 - 09/23

### Additional information

<img src="https://user-images.githubusercontent.com/108326843/191965570-cc77f3d8-eb8a-4818-95af-0ae90b233bc4.JPEG" width="50%" />

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21647)

**Comment by cpeterso** [comment]
Not a blocker for MR v106, but we should fix soon.

**Comment by AdinaPetridean** [comment]
Verified on Firefox Nighlty 107.0a1 - 10/06 with Xperia 10(Android 12).
